### ISSUE TITLE
Fix Proposal for Fine Offset High Pressure Offset issue.

### ISSUE THREAD

**Issue by bvs79669** [mcrossley/CumulusMX#1]
I found a fix for the high pressure offest issue with some Fine Offset weather stations. I tested the changes with my Station and it works very well. Please can you check and commit these changes?

@FoStation.cs:

// Suppress unused 2 highest bits on data [1] with 0x3F. This fixes wrong high pressure offets.
double relpressure = (((data[1] & 0x3F) * 256) + data[0]) / 10.0f;  

This is related to https://cumulus.hosiene.co.uk/viewtopic.php?f=27&t=16723

**Comment by Jasonkingsmill** [comment]
> I found a fox for the high pressure offest issue with some Fine Offset weather stations. I tested the changes with my Station and it works very well. Please can you check and commit these changes?
> 
> @FoStation.cs:
> 
> // Suppress unused 2 highest bits on data [1] with 0x3F. This fixes wrong high pressure offets.
> double relpressure = (((data[1] & 0x3F) * 256) + data[0]) / 10.0f;
> 
> This is related to https://cumulus.hosiene.co.uk/viewtopic.php?f=27&t=16723

I'm not familiar with the long-term issues...what are the typical ranges observed when it spikes? 

Given the realistic pressure ranges expected, the 2 most significant bits would make a reading non-sensical anyway.
I suspect, given that the memory map has been reverse engineered, those bits could possibly be unrelated to pressure anyway. I see no harm in pulling it in.

**Comment by bvs79669** [comment]
Here is some data from my station to show you how often the problem appeared. Look at the gaps of my pressure graph => 6 times in 2018:
https://www.wunderground.com/personal-weather-station/dashboard?ID=IBADENWR86#history/s20171230/e20181231/myear

For example, a value I've got is 2648,4 hPa in the Culumus MXDiag log and 1009,8 h Pa at the station's display at the same time. So I looked at the 
Hex values of 10098 -> 2772h
and                26484 -> 6774h -> Cleared the highest to bits. -> 2774h !!
And if I know that the maximum pressure value must be lower than 1200 hPa -> 12000 fits in 14-Bits everything is fine.

I'm using the fix since yesterday for my IBADENWR86 station and all pressure values are identical compared to my station's display.
